e2e/test/scenarios/admin/databases/database-prompt-banner.cy.spec.js

```diff
@@ -53,6 +55,74 @@ describe("banner", () => {
         ).should("not.exist");
       });
   });
+
+  describe("embeddings", () => {
+    const QUESTION_ID = 1;
+    it("should not render for public links", () => {
+      cy.request("POST", `/api/card/${QUESTION_ID}/public_link`, {}).then(
```

Comment:
There is `visitPublicQuestion` that does the same

Reply:
Fixed